catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -60,6 +60,20 @@ define(['underscore',
                 }
 
                 var provider = new type(initObj);
+
+                /*
+                  Optionally add this provider as a TrustedServer. This sets withCredentials = true
+                  on the XmlHttpRequests for CORS.
+                 */
+                if (model.get('withCredentials')) {
+                  var parsedUrl = new URL(provider.url);
+                  var port = parsedUrl.port;
+                  if (!port) {
+                    port = (parsedUrl.protocol === "https") ? 443 : 80;
```

Comment:
Where are these numbers coming from?

Reply:
They are just the standard ports for https and http. Cesium won't let you add a trusted server without a specific port.